.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* conda/conda-core
+* conda/steering-council
```

Comment:
Not sure why this is invalid; [the team exists](https://github.com/orgs/conda/teams/steering-council/members).